server/routerlicious/packages/lambdas/src/alfred/interfaces.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IEvent } from "@fluidframework/common-definitions";
+import { IClient, IConnected } from "@fluidframework/protocol-definitions";
+
+/**
```

Comment:
Thank you for documenting these interfaces! While we're here, it would be great to document their properties too if reviewers can add context 🙂